### PR TITLE
fixed styling on ipad and changed text

### PR DESCRIPTION
**Pull request by matejoslav**
https://trello.com/c/aQUqy9pH/213-traumatic-events-section-looks-bad-on-tablet

https://trello.com/c/xcHVw6TN/165-traumatic-events-text-in-taking-the-test-page

### CODE REVIEW

**Comment by MadelinTom** [comment]
Is the paragraph here meant to be aligned to the left? 
image

**Comment by matejoslav** [comment]
Will ask Tag

**Review by nicholasmorgan — CHANGES_REQUESTED** [review]
Needs fixing on mobile view

**Comment by MadelinTom** [comment]
FeelingSection.js line 28: max-width 90% here would make the paragraph the same width as the header text. We should ask design what they want this to look like.

File: web-app/src/components/FeelingsSection.js [hunk 1]
```diff
@@ -38,6 +38,14 @@ const Wrapper = styled.div`
   margin: 0 16.7%;
```
Comment (MadelinTom) [inline web-app/src/components/FeelingsSection.js:38]: We need to remove width: 50% here and make it maybe width 95% to match designs:

image

**Review by MadelinTom — CHANGES_REQUESTED** [review]
See comment

**Review by nicholasmorgan — APPROVED** [review]

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210307-135930 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210307-135930") to [dev](https://console.bluemix.net/apps/2ecd7e18-16aa-47d3-8830-a9d5742686d3?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 4569a14c6cf9a852180fe34485f777917f83a8e9

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210307-165809 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210307-165809") to [test](https://console.bluemix.net/apps/24633070-15da-462f-b17f-f8e232e0b152?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 4569a14c6cf9a852180fe34485f777917f83a8e9

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210307-175452 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210307-175452") to [prod](https://console.bluemix.net/apps/60de5256-5a33-4b93-8cb2-8eb23fcb0f7b?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:prod"), including commit 4569a14c6cf9a852180fe34485f777917f83a8e9

**Pull request merged**